pkg/rngd/fix-textrels-on-PIC-x86.patch

```diff
@@ -1,17 +1,17 @@
 --- rng-tools/rdrand_asm.S
 +++ rng-tools/rdrand_asm.S
 @@ -49,6 +49,7 @@
- 	ret
+	ret
```

Comment:
ah, looks like GH's web interface doesn't grok patch files very well. The changes are just whitespace changes, so should be fine